licenses/tests/factories.py

```diff
@@ -1,6 +1,11 @@
 import factory.fuzzy
 
-from licenses.models import LegalCode, License
+from licenses.models import LegalCode, License, TranslationBranch
+
+# The language codes we already have translations for
+language_codes = "ar,cs,de,el,en,es,eu,fi,fr,hr,id,it,ja,ko,lt,kv,mi,nl,no,pl,pt,ro,ru,sl,sv,tr,uk,zh_Hans,zh_Hant".split(
```

Comment:
should we include this line in our `constants.py` file and import `CURRENT_TRANSLATED_LANGUAGE_CODES` or something to that effect?

Reply:
Most of the code actually queries the database and knows exactly which languages we have. This is just for tests, which don't have real data to look at, but could fail if we try to use a language code without a pofile.